python/numpydoc.rst

```diff
@@ -636,6 +636,11 @@ The type inside the brackets is the type of each item.
 
    Leaving out the space results in a parsing error.
 
+.. warning::
+
+   The parameter name and type description must fit on a single line.
```

Comment:
This is not correct. As you can see in lsst/daf_butler#840 you can escape the line with a backslash. I only realized this last week when I saw @TallJimbo do it in some code.

Reply:
Confirmed; I'll do a few more experiments and document the behavior.